android/app/build.gradle

```
@@ -134,72 +93,13 @@ android {
 
     compileSdkVersion rootProject.ext.compileSdkVersion
 
+    namespace "com.bitrisereactnativesample"
```

Namespaces are supposed to use [reverse domain name notation](https://en.wikipedia.org/wiki/Reverse_domain_name_notation). As such, ours should start with `io.bitrise.`. I'd further suggest that we organize ourselves a bit:

```suggestion
    namespace "io.bitrise.sampleapps.reactnative"
```

Same for the applicationId and anywhere else you encounter `com.bitrise`.

---

Tried updating this and started getting build errors. Need to investigate further all the places that need to be changed.

---

Will create an issue and work on this tech debt separately.